.github/workflows/go-release.yml

```
@@ -112,12 +112,20 @@ jobs:
           pushd release
           helm package ../charts/${{ env.CHART_NAME }}/
           popd
+      - name: generate changelog
+        run: |
+          echo "# Cluster Proxy ${{ needs.env.outputs.RELEASE_VERSION }}" > /home/runner/work/changelog.txt
       - name: publish release
-        uses: ncipollo/release-action@v1
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          tag: ${{ env.RELEASE_VERSION }}
-          artifacts: "go/src/open-cluster-management.io/cluster-proxy/release/*.tgz"
-          token: ${{ secrets.GITHUB_TOKEN }}
+          body_path: /home/runner/work/changelog.txt
+          files: |
+            release/*.tgz
```

```suggestion
            go/src/open-cluster-management.io/cluster-proxy/release/*.tgz
```

since the default working-directory is go/src/open-cluster-management.io/cluster-proxy, https://github.com/open-cluster-management-io/cluster-proxy/pull/179/files#diff-29d508c8d0fc8d2fc3ea8e21d26ed76072f2de9d4a031e3307d41711e219bb76R16

---

Good point, I removed the default path to align with other repo and be simpler